static/index.html

```diff
@@ -6,9 +6,13 @@
     <title>Index</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="css/main.css" />
-    <script src="js/main.js"></script>
+    <script src="./js/main.js"></script>
+
 </head>
 <body>
-    
+    <div id='divtitle'>Sparkies Awesome Funtime UI</div>
+    <div id = 'mainb'>
+        <button onclick="begin()">Begin</button>
+    </div>
```

Comment:
Instead of making a button that starts the process, consider using [onload](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload)